config.py

```diff
@@ -0,0 +1,31 @@
+"""
+STATIC and SECRET data
+"""
+from pydantic import BaseSettings
+
+from functools import lru_cache
+
+
+class Settings(BaseSettings):
```

Comment:
**Static Code Issue**: Missing docstring in public class Settings. <br> **Fix**: Add a docstring to the Settings class. <br> **Code Suggestion**: 
 ```
 +    """Class to handle application settings from environment variables."""
 ```